src/main/java/org/folio/inventory/validation/experimental/InProcessNonRequestableTargetValidator.java

```diff
@@ -0,0 +1,26 @@
+package org.folio.inventory.validation.experimental;
+
+import org.folio.inventory.domain.items.Item;
+import org.folio.inventory.domain.items.ItemStatusName;
+
+import java.util.Set;
+
+import static java.util.Set.of;
+
+public class InProcessNonRequestableTargetValidator extends AbstractTargetValidator {
+
```

Comment:
Please remove this redundant blank line, I don't think it improves the readability of the code